fe/fe-core/src/main/java/com/starrocks/lake/LakeTable.java

```diff
@@ -164,11 +164,13 @@ public Status createTabletsForRestore(int tabletNum, MaterializedIndex index, Gl
                                           long partitionId, long shardGroupId) {
         FilePathInfo fsInfo = getPartitionFilePathInfo();
         FileCacheInfo cacheInfo = getPartitionFileCacheInfo(partitionId);
-
+        Map<String, String> properties = new HashMap<>();
+        properties.put(LakeTablet.PROPERTY_KEY_PARTITION_ID, Long.toString(partitionId));
+        properties.put(LakeTablet.PROPERTY_KEY_INDEX_ID, Long.toString(index.getId()));
```

Comment:
put table id

Reply:
Cannot get table id here, leaving this property unset is acceptable for now.